daml-foundations/daml-tools/daml-extension/syntaxes/daml12.tmLanguage-update.xml

```diff
@@ -1,89 +1,84 @@
-<!-- Copyright (c) 2017 Justus Adam -->
-
-<!-- All rights reserved. -->
-
-<!-- Redistribution and use in source and binary forms, with or without -->
-<!-- modification, are permitted provided that the following conditions are met: -->
-
-<!--     * Redistributions of source code must retain the above copyright -->
-<!--       notice, this list of conditions and the following disclaimer. -->
```

Comment:
Are violating this rule by removing the copy right header?

Reply:
Yep, let's leave this

Reply:
this is been taken off from the haskell vs code plugin grammar file https://github.com/JustusAdam/language-haskell/blob/master/syntaxes/haskell.tmLanguage